pandas/core/generic.py

```diff
@@ -3916,8 +3903,14 @@ def _take(
                 and is_range_indexer(indices, len(self))
             ):
                 return self.copy(deep=None)
+        elif self.ndim == 1:
+            # TODO: be consistent here for DataFrame vs Series
```

Comment:
@phofl following this ill do do the deprecation we discussed

Reply:
Sounds good!